schema.graphql

```diff
@@ -2,26 +2,28 @@ type Contribution @entity {
   id: ID!
 
   paraId: Int! @index
-  vaultId: String!
+  vault: Vaults! @index
   blockHeight: Int! @index
   account: String! @index
   amount: String!
-  referralCode: String
   timestamp: Date!
 }
 
 type Vaults @entity {
   id: ID!
-
-  createdAt: String
+  createdAt: Int
   paraId: Int! @index
-  vaultId: String!
   ctokenId: Int!
   phase: String!
   contributions: Int!
   totalAmount: String!
   contributionStrategy: String!
   cap: String!
   endBlock: Int!
-  trieIndex: Int!
+}
+
+type VaultsDissolve @entity {
+  id: ID!
+  vault: Vaults! @index
```

Comment:
If we have the same vaultId for one project with differernt status, one is dissolved, one is started, how to figure out it?